### DISCUSSION TITLE
What is `qstat -x`?

### DISCUSSION THREAD

**Discussion by HuangJiameng** [deepmodeling/dpdispatcher#250] [Q&A]
https://github.com/deepmodeling/dpdispatcher/blob/master/dpdispatcher/pbs.py#L55
I can't find the meaning of `qstat -x` on any website. Does anyone know if it is wrong here?

**Comment by njzjz** [comment]
It can be found in the source code: https://github.com/openpbs/openpbs/blob/b4b1254368f43c1d9baf48770a83384f08498647/src/cmds/qstat.c#L2477
